utils_lib/src/cameligo/contracts/utils.mligo

```diff
@@ -11,6 +11,59 @@ module Address = struct
 
 end
 
+module Math = struct 
+
+    let sqrt (y: nat) =
```

Comment:
should have comments specifying that it gives the integer square root https://en.wikipedia.org/wiki/Integer_square_root

Reply:
and be named `isqrt` like in the wikipedia page